packages/eslint-config/package.json

```diff
@@ -17,13 +17,14 @@
     "typescript": ">=4.2"
```

Comment:
We should be removing the peer dependencies here, just like in the other package.